lib/teamsnap.rb

```diff
@@ -34,6 +35,15 @@ def init(opts = {})
       ##   create universally accessible TeamSnap.root_client
       self.root_client = TeamSnap::Client.new(:token => token)
 
+      ##   include any feature headers
+      if opts[:header]
```

Comment:
Passing in multiple headers would make sense, or are you think it would?

```
client = TeamSnap.init(
	:url => ROOT_TEST_URL,
	:client_id => client_id,
	:client_secret => client_secret,
	:header => {"X-Teamsnap-Api-Features" => "ghost_contact", "User-Agent", "anexampleheadervalue"}
)
```